identity/aziot-identityd/src/lib.rs

```diff
@@ -429,192 +396,76 @@ impl Api {
-                            sas_key: symmetric_key.clone(),
-                        };
-                        let credential = aziot_identity_common::Credentials::SharedPrivateKey(
-                            symmetric_key.clone(),
-                        );
-
-                        let operation = dps_client
-                            .register(&registration_id, &dps_auth_kind)
-                            .await
-                            .map_err(Error::DPSClient)?;
-
-                        operation_to_iot_hub_device(credential, operation).await?
-                    }
-                    config::DpsAttestationMethod::X509 {
-                        registration_id,
-                        identity_cert,
-                        identity_pk,
-                    } => {
-                        self.create_identity_cert_if_not_exist_or_expired(
-                            &identity_pk,
-                            &identity_cert,
-                            &registration_id,
-                        )
-                        .await?;
-
-                        let dps_auth_kind = aziot_dps_client_async::DpsAuthKind::X509 {
-                            identity_cert: identity_cert.clone(),
-                            identity_pk: identity_pk.clone(),
-                        };
-                        let credential = aziot_identity_common::Credentials::X509 {
-                            identity_cert: identity_cert.clone(),
-                            identity_pk: identity_pk.clone(),
-                        };
-
-                        let operation = dps_client
-                            .register(&registration_id, &dps_auth_kind)
-                            .await
-                            .map_err(Error::DPSClient)?;
-
-                        operation_to_iot_hub_device(credential, operation).await?
-                    }
-                    config::DpsAttestationMethod::Tpm { registration_id } => {
-                        let dps_auth_kind = aziot_dps_client_async::DpsAuthKind::Tpm;
-                        let credential = aziot_identity_common::Credentials::Tpm;
-
-                        let operation = dps_client
-                            .register(&registration_id, &dps_auth_kind)
-                            .await
-                            .map_err(Error::DPSClient)?;
-
-                        operation_to_iot_hub_device(credential, operation).await?
-                    }
-                };
-                self.id_manager.set_device(&device);
-                aziot_identity_common::ProvisioningStatus::Provisioned(device)
-            }
-            config::ProvisioningType::None => {
-                log::info!("Skipping provisioning with IoT Hub.");
+        log::info!("Provisioning starting.");
 
-                aziot_identity_common::ProvisioningStatus::Unprovisioned
+        let _ = match trigger {
```

Comment:
Log messages when settings change (with the reason) would be helpful while debugging.

Reply:
Fixed